gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Jul 27 16:18:33 CEST 2015
+#Fri Jul 10 10:40:08 MSK 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
```

Comment:
If you've loaded this in IntelliJ, it may have asked for the source distribution so it can do completions better. That's done by changing this `bin` to an `all` or by doing some [configuration to the wrapper task](https://docs.gradle.org/6.4.1/userguide/gradle_wrapper.html#customizing_wrapper).

Reply:
Yep it did that for me. Left it in for my commit.